Local LDAP server

buchi opened the pull request:
Add a local LDAP server that is run with Docker Compose.
This way we no longer depend on connectivity to the 4teamwork LDAP.

## Checklist

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [ ] Changelog entry
- [ ] Link to issue (Jira or GitHub) and backlink in issue (Jira)

lukasgraf reviewed (approved):
LGTM! 👍 Just some small comments.
Thanks for the extensive updates to the README!

buchi commented on docker-compose.yml:
```diff
@@ -94,12 +94,14 @@ services:
       - LDAP_USER_BASE_DN=${LDAP_USER_BASE_DN}
       - LDAP_GROUP_BASE_DN=${LDAP_GROUP_BASE_DN}
+      - 'USER_ATTRIBUTE_MAPPING={"external_id":"entryUUID","username":"uid","firstname":"givenName","lastname":"sn","email":"mail","display_name":"displayName","organization":"o","department":"ou"}'
+      - 'GROUP_ATTRIBUTE_MAPPING={"external_id":"entryUUID","groupname":"cn","title":"description"}'
```
Removed mapping

buchi commented on README.rst:
```diff
@@ -3,169 +3,62 @@ opengever.core
-HAProxy
-^^^^^^^
+    $ cp .env.sample .env
+    $ docker compose up -d
```
This is not necessary. The database is created automatically on startup of the ogds container.

buchi commented:
I've now added default env values in the compose file. A `.env` file is no longer needed, but can be used to customize settings.

Removed employee user names from example content.
